src/ripple/protocol/impl/STValidation.cpp

```diff
@@ -161,6 +161,7 @@ SOTemplate const& STValidation::getFormat ()
             format.push_back (SOElement (sfSigningTime,     SOE_REQUIRED));
             format.push_back (SOElement (sfSigningPubKey,   SOE_REQUIRED));
             format.push_back (SOElement (sfSignature,       SOE_OPTIONAL));
+            format.push_back (SOElement (sfConsensusHash,   SOE_OPTIONAL));
         }
     };
 
```

Comment:
While we're in here maybe we can make `holder` on line 168 `const`.  Perhaps something like `static FormatHolder const holder;`?